src/components/akoma-ntoso/akoma-ntoso.scss

```diff
@@ -16,4 +15,16 @@ la-akoma-ntoso {
       animation: fadeout 2s;
     }
   }
+
+  &[frbr-country="ke"] {
```

Comment:
These selectors need to be `&[frbr-country^="ke"]` so that KE styles are applied to `frbr-country="ke"` as well as `frbr-country="ke-xxx"` (ie. a place within KE).